packages/react-components/react-field/.storybook/main.js

```diff
@@ -2,7 +2,12 @@ const rootMain = require('../../../../.storybook/main');
 
 module.exports = /** @type {Omit<import('../../../../.storybook/main'), 'typescript'|'babel'>} */ ({
   ...rootMain,
-  stories: [...rootMain.stories, '../stories/**/*.stories.mdx', '../stories/**/index.stories.@(ts|tsx)'],
+  stories: [
```

Comment:
Should these `react-field` changes be in this PR? Based on your description I wouldn't expect to see these.

Reply:
ditto to my previous comment. This is a nistake. Thanks for catching!